packages/flutter/lib/src/widgets/restoration_properties.dart

```diff
@@ -185,6 +228,35 @@ class _RestorablePrimitiveValue<T extends Object> extends RestorableValue<T> {
   }
```

Comment:
Can you also update the comment above _RestorablePrimitiveValue that explains how we could introduce nullable versions and make it point to _RestorablePrimitiveValueN?